tests/virt_autotest/libvirt_isolated_virtual_network.pm

```diff
@@ -27,17 +27,21 @@ sub run_test {
     virt_autotest::virtual_network_utils::download_network_cfg($vnet_isolated_cfg_name);
 
     #Stop named.service, refer to poo#175287
-    systemctl("stop named.service") if (is_sle('>=15-SP6') && check_var('VIRT_AUTOTEST', 1));
+    systemctl("stop named.service") if (is_sle('>=15-SP6') && is_sle('<16') && check_var('VIRT_AUTOTEST', 1));
```

Comment:
For this named workaround, I remember in the relevant bug there is solution by updating configuration file of named. Better to make it into code sooner :), but not necessarily in this PR ofc.